bundles/org.openhab.binding.network/README.md

```diff
@@ -62,6 +62,7 @@ Use the following options for a **network:speedtest**:
 -   **uploadSize:** Size of the file to be uploaded in bytes. Default is 1000000.
 -   **url:** Url of the speed test server.
 -   **fileName:** Name of the file to download from test server
+-   **initialDelay:** Delay (in minutes) before starting the first speed test (can help avoid flooding your server at startup). Default : 5.
```

Comment:
Hey @clinique,
this part of the Readme needs some corrections. The "Default" part is given in three different formats and not indicated for two parameters. Then there is this mistake in refreshInterval: "in minutes. Default: `60000` (one minute)"
Do you want to clarify and/or create a PR?